devtools/conda-envs/dev_head.yaml

```diff
@@ -0,0 +1,34 @@
+name: qcarchive
+channels:
+  - defaults
+  - conda-forge
+dependencies:
+  - python
+  - numpy
+  - pandas
+  - mongodb
+  - pymongo
+  - mongoengine
+  - tornado
+  - requests
+  - jsonschema
+  - bcrypt
+  - cryptography
+
+# Test depends
+  - pytest
+  - pytest-cov
+  - codecov
+
+# Environment specific includes
+  - rdkit
+
+# QCArchive includes
+
+# Pip includes
+  - pip:
+    - pydantic
+    - git+git://github.com/MolSSI/QCEngine#egg=qcengine
+    - git+git://github.com/MolSSI/QCElemental#egg=qcengine
```

Comment:
I don't know this syntax, but should it be `egg=qcelemental`?

Reply:
That could help yes :)